mlrun/serving/states.py

```diff
@@ -1208,6 +1209,7 @@ def add_model(
         self,
         endpoint_name: str,
         model_class: str,
+        execution_mechanism: str,
```

Comment:
Why not provide a default value here? This will force any user adding any model to provide a mechanism. It could default to `naive`, or any other mechanism that we decide makes sense by default.

Reply:
I don't think there is a default value we can choose without creating a pitfall for the user, though.